app/models/autocomplete_locations/autocomplete_location.rb

```diff
@@ -12,5 +12,23 @@ def self.city_name_ids(id)
       same_city_names(id).select(:id)
     end
 
+    def is_default_city?
+      searched_city = "#{self[:city]}, #{AutocompleteLocations::State.state_name(self[:state])}"
+      AutocompleteLocations.current_city_name?(searched_city) == self[:city]
+    end
+
+    def self.default_city(id)
+      default_city = AutocompleteLocation.find_by(city_id: AutocompleteLocation.find(id).city_id, default_city: true)
+      default_city || AutocompleteLocation.find(id)
+    end
+
+    def self.current_city_name?(query)
+      # Try query with city name and state both. Ex: query = 'Aurangabad, Uttar Pradesh'
+      searched_city = Geocoder.search query
```

Comment:
Might need to consider hard dependency on `Geocoder` here for this task specifically.

Reply:
Also, this change was specifically required for our DB, might not be relevant generally. I would suggest looking at it and closing this PR is irrelevant.

Reply:
Does it not also handle cases where a city was renamed?

Reply:
yes, this has hard dependency on `geocoder`, and I found out they keep changing their API.
I'll see what else can be the solution for finding out which should be the current default city.
Like how to decide if `Bangalore` or `Bengaluru` should be used.